### PR TITLE
Lesson 1

### PR DESCRIPTION
**Pull request by 503292**

### CODE REVIEW

File: Lesson-1/src/routes/router.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const mainRoute = require("./main/main");
+const motocycleRoute = require("./motocycle/motocycle");
+const signUpRoute = require("./users/sign-up-route");
+const productsRoute = require("./products/products-route");
+
+const router = {
+  "/signup": signUpRoute,
+  "/motocycle": motocycleRoute,
```
Comment (odykyi) [inline Lesson-1/src/routes/router.js]: remove motocycle in all project

File: Lesson-1/src/routes/users/sign-up-route.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+const qs = require("querystring");
+const path = require("path");
+const fs = require("fs");
+
+const saveUser = user => {
+  username = JSON.parse(user);
```
Comment (odykyi) [inline Lesson-1/src/routes/users/sign-up-route.js]: not username

Rename

File: Lesson-1/src/routes/users/sign-up-route.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  const pathToFile = path.join(
+    __dirname,
+    "../../",
+    "db",
+    "users",
+    `${username.username}.json`
```
Comment (odykyi) [inline Lesson-1/src/routes/users/sign-up-route.js]: username.username ?

File: Lesson-1/src/routes/products/products-route.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+const fs = require("fs");
+const path = require("path");
+
+const productsRoute = (request, response) => {
+  let filePath = path.join(
```
Comment (odykyi) [inline Lesson-1/src/routes/products/products-route.js]: why not const?

File: Lesson-1/src/routes/products/products-route.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    "all-products.json"
+  );
+
+  const allProducts = fs.readFileSync(filePath, "utf8");
+
+  response.writeHead(200, {
+    "Content-Type": "application/json"
+  });
+
+  let file = JSON.parse(allProducts);
```
Comment (odykyi) [inline Lesson-1/src/routes/products/products-route.js]: why not const?

File: Lesson-1/src/routes/demo/demo-with-dir.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+const fs = require("fs");
+
```
Comment (odykyi) [inline Lesson-1/src/routes/demo/demo-with-dir.js]: remove demo form project

File: Lesson-1/src/routes/router.js (hunk 1 above)
Comment (503292) [reply Lesson-1/src/routes/router.js]: Done

File: Lesson-1/src/routes/users/sign-up-route.js (hunk 2 above)
Comment (503292) [reply Lesson-1/src/routes/users/sign-up-route.js]: done

File: Lesson-1/src/routes/users/sign-up-route.js (hunk 3 above)
Comment (503292) [reply Lesson-1/src/routes/users/sign-up-route.js]: done

File: Lesson-1/src/routes/demo/demo-with-dir.js (hunk 6 above)
Comment (503292) [reply Lesson-1/src/routes/demo/demo-with-dir.js]: done

File: Lesson-1/src/routes/products/products-route.js (hunk 5 above)
Comment (503292) [reply Lesson-1/src/routes/products/products-route.js]: done

File: Lesson-1/src/routes/products/products-route.js (hunk 4 above)
Comment (503292) [reply Lesson-1/src/routes/products/products-route.js]: done

**Comment by 503292** [comment]
FIX _ all reviews

**Pull request closed without merging**